frontend/src/app/testResults/mocks/queries.mock.tsx

```diff
@@ -30,6 +30,21 @@ import resultForViewing from "./resultForViewing";
 import resultForCorrection from "./resultForCorrection";
 
 export const mocks = [
+  {
```

Comment:
is there a reason this mock needed to get added?

Reply:
The TestResultList.test.tsx was throwing an apollo warning about that request missing a mock so I added it :D